dohq_artifactory/utils.py

```diff
@@ -0,0 +1,6 @@
+import logging
```

Comment:
I do not like "utils" "helpers" modules, they may become too heave. 
Let's call it `logging.py` or even `logger`

It does one thing and do it well

Reply:
@allburov 
done